### ISSUE TITLE
Car damage incorrect?

### ISSUE THREAD

**Issue by JepsonHepson** [InZidiuZ/op-framework-issue-tracker#871]
**Reporter:**
@Jepson#1379
https://steamcommunity.com/id/Di-Deli/

**Summary:**
I don't believe the damage numbers for the cars are correct.
After some testing, it took 310 AR rounds to disable a local vehicle. I remember a poll made a bit back https://gyazo.com/a86783a5ce3236833e673a0c28d33114, however this 310 bullet count doesn't really add up. This was tested on multiple vehicles. Testing was done on PD vehicles as well and it seemed to yield similar results.

**Steps to Reproduce:**
Shoot a car at whichever distance. 

**Expected Results:**
For it to break a lot sooner.

**Actual Results:**
It becomes a tank. (without any armor upgrades from LSC)

**Comment by InZidiuZ** [comment]
Were there a player behind the wheel in the local vehicle?

**Comment by Duroode** [comment]
> Were there a player behind the wheel in the local vehicle?

Yes, He was Behind the wheel, i was shooting at the engine block.

**Comment by JepsonHepson** [comment]
From what I've seen is that the damage from crashes in general are lowered as well. I have seen crashes at top speed, nothing really happened to the car, barely a dent.

**Comment by Doughnut2001** [comment]
yeah, i've seen this happen, cars start smoking for a bit but then the engine just keeps going like nothing happened, it'd be nice to see some systems impemented that actually make you drive safely so you don't end up going 200 miles down elgin and not caring about crashing at all

**Comment by JepsonHepson** [comment]
This is really a big problem, cars never disable, not even through crashing or whatnot, everything is literally a tank and it's really fucking with a lot of stuff

**Comment by Nuziatella** [comment]
Agreed that this is an issue. Saw a car going about 100+mph crash into a concrete wall and then reversed and took off again. Should result in serious damage to the car with perhaps even popped tires and the like as well as serious damage to the passengers/driver in the vehicle.
